### PR TITLE
ROX-11558: Run fleetshard-operator on dev/CI environments

### PR DESCRIPTION
**Pull request by kovayur**
## Description
Run fleetshard-operator on dev/CI environments

## Checklist (Definition of Done)

- [x] Unit and integration tests added
- [x] Added test description under `Test manual`
- [ ] Documentation added if necessary (i.e. changes to dev setup, test execution, ...)
- [x] CI and all relevant tests are passing
- [x] Add the ticket number to the PR title if available, i.e. `ROX-12345: ...`
- [ ] Discussed security and business related topics privately. Will move any security and business related topics that arise to private communication channel.
- [ ] Add secret to app-interface Vault or Secrets Manager if necessary
- [ ] RDS changes were e2e tested [manually](../docs/development/howto-e2e-test-rds.md)
- [ ] Check AWS limits are reasonable for changes provisioning new resources
- [ ] (If applicable) Changes to the dp-terraform Helm values have been reflected in the addon on integration environment

## Test manual
E2E tests

### CODE REVIEW

File: e2e/e2e_test.go [hunk 1]
```diff
@@ -57,10 +57,6 @@ var _ = Describe("Central", Ordered, func() {
 	var notes []string
 	var ctx = context.Background()
 
-	BeforeEach(func() {
```
Comment (kovayur) [inline e2e/e2e_test.go:60]: This negates the "should install ForceReconcile via gitops" step.

File: dev/env/scripts/bootstrap.sh [hunk 2]
```diff
@@ -49,13 +49,6 @@ else
     log "Skipping installation of Vertical Pod Autoscaler"
 fi
 
-if [[ "$INSTALL_VERTICAL_POD_AUTOSCALER_OLM" == "true" ]]; then
```
Comment (kovayur) [inline dev/env/scripts/bootstrap.sh:52]: Superseded by the helm chart

File: e2e/e2e_test.go [hunk 3]
```diff
@@ -365,7 +365,7 @@ var _ = Describe("Central", Ordered, func() {
 				Should(BeEmpty(), "Started at %s", time.Now())
 		})
 
-		It("should restore the default gitops config", func() {
+		AfterAll(func() {
```
Comment (kovayur) [inline e2e/e2e_test.go:368]: Execute even if tests failed

**Review by kurlov — APPROVED** [review]

**Comment by kurlov** [comment]
I didn't have an opportunity to run locally on my machine

**Pull request merged**